lib/mods/replace-get-set.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const nodeTypes = require('ts-morph').TypeGuards;
+const {updateImports} = require('../imports.js');
+const {
+  isGlobal,
+  getOriginal,
+} = require('../originals-helper.js');
+const {formatCall} = require('./helpers.js');
+
+/**
+ * Replaces accessor calls in the current node with the Get-Originals version
+ *
+ * @param {Node} node - a ts-morph AST Node
+ * @param {Object} imports - the current list of imports
+ */
+function replaceGlobalGetSetCalls(node, imports) {
+  if (nodeTypes.isPropertyAccessExpression(node) &&
+      !nodeTypes.isCallExpression(node.getParent()) &&
+      isGlobal(node.getExpression())) {
+    const method = node.getSymbol().getEscapedName();
+    const baseTypeName = node.getExpression().getType().getSymbol().getName();
+    const callee = node.getExpression().getSymbol().getName();
+    let type;
+    let args;
+
+    // Check if the expression is receiving assignment (set)
+    if (nodeTypes.isBinaryExpression(node.getParent()) &&
+        node.getParent().getLeft() === node) {
+      type = 'set';
+      args = node.getParent().getRight().getText();
+    } else {
+      type = 'get';
+    }
+
+    const original = getOriginal(method, type, baseTypeName);
+
+    let base;
```

Comment:
I don't think "base" is quite right here. You're potentially replacing a whole node, not just the base portion. `nodeToReplace` seems more accurate.